app/src/marketplace/containers/EditProductPage/index.jsx

```diff
@@ -35,6 +37,8 @@ import CropImageModal from './CropImageModal'
 
 import styles from './editProductPage.pcss'
 
+const STREAMS_PAGE_SIZE = 999
```

Comment:
IIUC - this is moreso a `MAX_FETCH_SIZE` now?

Reply:
Yes and no, I guess `MAX_FETCH_SIZE` would be 1000 but we are only fetching one below that